src/plugins/ThumbnailGenerator/index.test.js

```diff
@@ -329,5 +329,38 @@ describe('uploader/ThumbnailGeneratorPlugin', () => {
         getContext: canvas.getContext
       })
     })
+
+    it('should upsize if original image is smaller than target size', () => {
+      const core = {}
+      const plugin = new ThumbnailGeneratorPlugin(core)
+      const image = {
+        width: 100,
+        height: 80
+      }
+      plugin.downScaleInSteps = jest.fn().mockReturnValue({
+        image: {
+          height: 160,
+          width: 200
+        },
+        sourceWidth: 200,
+        sourceHeight: 160
+      })
```

Comment:
the `downScaleInSteps` method was removed recently so I don't think it's necessary to mock it here—we should also remove it from the other tests